### ISSUE TITLE
[BUG] Icon is Broken under KDE Plasma

### ISSUE THREAD

**Issue by polluxau** [Vysp3r/ProtonPlus#260]
**Describe the bug**
Icon looks extremely white or overexposed in discover and when the shortcut is created

**To Reproduce**
1. Download protonplus on kde plasma
2. Look at the icon that is broken

**Expected behavior**
Should look correct

**Screenshots**
![image](https://github.com/user-attachments/assets/651dfab2-b192-4acd-8444-2269b196a4ed)

![image](https://github.com/user-attachments/assets/82b6c88f-d5d4-40ce-b1bb-8d538db131e1)

**Desktop (please complete the following information):**
 - OS:Bazzite
 - DE: KDE Plasma 6.1.5
 - Type: Flatpak

**Additional context**

**Comment by Arcitec** [comment]
I see. So KDE is not able to render SVG icons correctly.

Shame on KDE.

We will have to generate PNG files for KDE users then. But Vysp3r is gonna be gone for over a month, and I can't create releases without him, so it will have to wait. We will take care of it later. Thank you for reporting it. :)

**Comment by Vysp3r** [comment]
I just pushed an update to fix that issue

**Comment by polluxau** [comment]
> I see. So KDE is not able to render SVG icons correctly.
> 
> Shame on KDE.
> 
> I see that it's rendering the rounded square background and its gradient correctly. But it's only rendering a single pixel of the overlay "P+" shape, lol.
> 
> Our SVG icon is correct. You can see the proper rendering by visiting it in your web browser here:
> 
> https://github.com/Vysp3r/ProtonPlus/blob/main/data/logo/com.vysp3r.ProtonPlus.svg
> 
> So it's entirely KDE's fault. Which means we will have to generate bitmap PNG icons for KDE users then. But Vysp3r is gonna be gone for over a month, and I can't create releases without him, so it will have to wait. We will take care of it later. Thank you for reporting it. :)
> 
> Edit: We worked hard to get it done before Vysp3r has to go. It's coming in a release today.

Thank you for the quick and hard work! It's quite confusing that KDE doesn't have good svg support by now :/

**Comment by Arcitec** [comment]
> Thank you for the quick and hard work! It's quite confusing that KDE doesn't have good svg support by now :/

Yeah I'm glad we got it done today.

And yeah, it was definitely a lesson to not rely on SVG rendering. Thought everyone on Linux would be using some standardized SVG library to draw it correctly on every desktop, but apparently that's not how it works. Simple SVG icons are probably okay on KDE, but we used gradients and drop shadows and some of that clearly broke KDE's renderer.

Oh well, now we have PNGs so it'll look the same everywhere. :)
